includes/Hooks.php

```diff
@@ -54,6 +61,9 @@
 use Throwable;
 use Wikimedia\IPUtils;
 use Wikimedia\Rdbms\ILBFactory;
+use Article;
```

Comment:
This needs to be reverted. CI requires no comments in use statements and that it be alphabetical order.